runtime/jcl/common/vm_scar.c

```diff
@@ -301,6 +301,17 @@ scarInit(J9JavaVM * vm)
 #if defined(J9VM_INTERP_MINIMAL_JCL)
 	result = standardInit(vm, J9_DLL_NAME);
 #else /* J9VM_INTERP_MINIMAL_JCL */
+#if defined(OSX) && (JAVA_SPEC_VERSION >= 16)
+	{
+		const char *osxsecurityStr = "osxsecurity";
```

Comment:
This library exists in jdk11 as well, does it not need the same fix?

Reply:
This only applies `JDK16+` because of `Panama` LibraryLookup support introduced by `Refactor native library loading implementation` [1]
JNI library loading in pre-JDK16 level works same as before.

[1] https://bugs.openjdk.java.net/browse/JDK-8228336